--- con/configure.orig	2024-07-06 21:09:12.645559573 +0800
+++ con/configure	2024-07-06 22:43:37.665557412 +0800
@@ -3168,10 +3168,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     STRIP=$ac_ct_STRIP
   fi
 else
@@ -3766,10 +3767,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     CC=$ac_ct_CC
   fi
 else
@@ -3986,10 +3988,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     CC=$ac_ct_CC
   fi
 fi
@@ -4088,10 +4091,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     CC=$ac_ct_CC
   fi
 else
@@ -5026,10 +5030,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     CXX=$ac_ct_CXX
   fi
 fi
@@ -6186,10 +6191,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     DUMPBIN=$ac_ct_DUMPBIN
   fi
 fi
@@ -6638,10 +6644,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     OBJDUMP=$ac_ct_OBJDUMP
   fi
 else
@@ -6993,10 +7000,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     DLLTOOL=$ac_ct_DLLTOOL
   fi
 else
@@ -7156,10 +7164,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     AR=$ac_ct_AR
   fi
 fi
@@ -7329,10 +7338,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     STRIP=$ac_ct_STRIP
   fi
 else
@@ -7438,10 +7448,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     RANLIB=$ac_ct_RANLIB
   fi
 else
@@ -8326,10 +8337,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     MANIFEST_TOOL=$ac_ct_MANIFEST_TOOL
   fi
 else
@@ -8457,10 +8469,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     DSYMUTIL=$ac_ct_DSYMUTIL
   fi
 else
@@ -8559,10 +8572,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     NMEDIT=$ac_ct_NMEDIT
   fi
 else
@@ -8661,10 +8675,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     LIPO=$ac_ct_LIPO
   fi
 else
@@ -8763,10 +8778,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     OTOOL=$ac_ct_OTOOL
   fi
 else
@@ -8865,10 +8881,11 @@
   else
     case $cross_compiling:$ac_tool_warned in
 yes:)
+    if ! echo "$CC" | grep -q "^llvm-"; then
 { printf "%s\n" "$as_me:${as_lineno-$LINENO}: WARNING: using cross tools not prefixed with host triplet" >&5
 printf "%s\n" "$as_me: WARNING: using cross tools not prefixed with host triplet" >&2;}
-ac_tool_warned=yes ;;
-esac
+ac_tool_warned=yes 
+esac ;;
     OTOOL64=$ac_ct_OTOOL64
   fi
 else
